build/lender/Lender.sol

```diff
@@ -18,19 +18,27 @@ contract Lender {
   address public pendleAddr;
   address public tempusAddr;
 
+  uint256 public feenominator;
+
   event Lend(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 returned);
   event Mint(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 amount);
 
   /// @param s: the swivel contract
   /// @param p: the pendle contract
   /// @param t: the tempus contract
-  constructor(address s, address p, address t) {
+  /// @param f: set to be the feenominator, is correlated to the fee rate
+  constructor(address s, address p, address t, uint256 f) {
     admin = msg.sender;
     swivelAddr = s;
     pendleAddr = p;
     tempusAddr = t;
+    feenominator = f;
```

Comment:
so `feenominator` was used in swivel because it was always the denominator in the calculation of the fee. if that's the case here then the name is good obvs.